src/Lumi/Components/Table.purs

```diff
@@ -77,7 +77,7 @@ type TableProps row =
         , filterLabel :: Nullable String
         , sortBy :: Nullable ColumnName
         , style :: R.CSS
-        , getLink :: row -> URL
+        , getLink :: Nullable (row -> URL)
```

Comment:
Is this Nullable (as opposed to Maybe) because it's still being used from JS?

Reply:
I think we've just recently migrated all the JS usage, so this is just matching the existing API until we change them all to Maybes